pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -411,6 +413,18 @@ func (p *PriorityQueue) isPodWorthRequeuing(logger klog.Logger, pInfo *framework
 	}
 
 	if event.IsWildCard() {
+		// If the wildcard event has a Pod in newObj,
+		// that indicates that the event wants to be effective for the Pod only.
+		// Specifically, EventForceActivate could have a target Pod in newObj.
+		if newObj != nil {
+			if pod, ok := newObj.(*v1.Pod); ok && pod.UID == pInfo.Pod.UID {
+				logger.V(6).Info("Worth requeuing because the event is wildcard", "pod", klog.KObj(pInfo.Pod))
```

Comment:
you can add "wildcard dedicated to this pod"

Reply:
I took macsko's suggestion and this path is no longer existing. But, I improved the log in a similar way.